DevelopersGuide.adoc

```diff
@@ -8057,7 +8056,7 @@ initial-form (f/validate-fields (f/build-form MyForm my-entity-props))
 ```
 
 The functions `valid?` and `invalid?` honor the tri-state nature of forms (e.g. `invalid?` returns
-true iff at least one field is `:invalid`, and `valid?` returns true iff all fields are
+true IFF at least one field is `:invalid`, and `valid?` returns true IFF all fields are
```

Comment:
IFF isn't an acronym, but it also isn't good form probably...should be "if and only if"

Reply:
maybe better to expand this one, I would have no idea what `iff` means.